src/configuration.ts

```diff
@@ -233,7 +233,7 @@ export class Configuration {
       version = env.GAE_MODULE_VERSION;
     }
 
-    this._serviceContext.service = isString(service) ? service : 'node';
+    this._serviceContext.service = (isString(service) ? service : 'node')!;
```

Comment:
nit: Maybe cast `isString` to `(obj: any) => obj is string` type?

Reply:
That would work too.  In the long run I want to remove the `is*` methods where possible.